README.md

```diff
@@ -0,0 +1,173 @@
+# Cloudstate Sample Shopping Cart Application
+
+## Sample application layout
+
+The sample application consists of 2 services:
+* A stateless service `frontend`
+* A stateful Entity based service `shopping-cart`
+
+Additionally:
+* A `deploy` directory that contains the deployment yaml files.
+
+### Quick Install
+
+All the latest docker images are available publicly at `lightbend-docker-registry.bintray.io/cloudstate-samples`.
+
+To deploy the shopping-cart application as is, connect to your kubernetes environment and do the following.
+
+```bash
+$ cd deploy
+$ kubectl apply -f . -n <project-name>
+# verify stateful store
+$ kubectl get -n <project-name> statefulstore
+NAME                  AGE
+shopping-store   21m
+# verify stateful services
+$ kubectl -n <project-name>  get statefulservices
+NAME            AGE    REPLICAS   STATUS
+shopping-cart   7m     1          Running
+frontend        4m     1          Running
+```
+
+`https://<project-name>.us-east1.apps.lbcs.dev/pages/index.html`
```

Comment:
Is this location meant to be in the OSS sample documentation? It seems to be here without context.